pydap_extras/app.py

```diff
@@ -0,0 +1,10 @@
+import sys
+from paste.httpserver import serve
+from pydap.wsgi.ssf import ServerSideFunctions
+from pydap_extras.handlers import csv
+
+if __name__ == "__main__":
+    csv._test()
+    application = csv.CSVHandler(sys.argv[1])
+    application = ServerSideFunctions(application)
+    serve(application, port=8001)
```

Comment:
This looks good but if we are planning to introduce other handlers this will need to be more robust. Just something to think about when we get to that point.

Reply:
Also we may want to detail how to run this in the readme as well as other setup instructions.